src/main/webapp/js/userMapData.json

```diff
@@ -1059,17 +1109,20 @@
     "STEM Launch",
     "Stephen F. Austin State University",
     "Stevens Institute of Technology",
+    "Suffolk University",
     "Sun Valley High School",
     "SUNY Buffalo",
     "SUNY Jefferson Community College",
     "Syracuse University",
+    "TAMC 6B2",
     "Tarleton State University",
     "Tarrant County College",
     "Taylor University",
     "Temple University",
     "Tennessee Technological University",
     "Texas A&M Health Science Center",
     "Texas A&M International University",
+    "Texas A&M of Corpus Christi",
```

Comment:
This should be duplicate of three entries after this: `Texas A&M University-Corpus Christi`.